xml/snmp_sysdescr.xml

```diff
@@ -1598,6 +1598,7 @@ Copyright (c) 1995-2005 by Cisco Systems
     <param pos="0" name="os.certainty" value="0.85"/>
     <param pos="0" name="os.vendor" value="Cisco"/>
     <param pos="0" name="os.device" value="Switch"/>
+    <param pos="1" name="os.family"/>
```

Comment:
Interesting -- I thought things like this were supposed to be caught by some of the recog unit tests.   Regular expressions that over/under capture are supposed to be flagged.